wandb/sdk/wandb_settings.py

```diff
@@ -462,6 +462,9 @@ class Settings(BaseModel, validate_assignment=True):
     username: Optional[str] = None
     """Username."""
 
+    wandb_rel_dir: Optional[str] = None
```

Comment:
Maybe `wandb_dir_name`? Or even `wandb_stage_dir`?

We also may want to have validation preventing this from being a multi-component relative path, since that'd be redundant with `root_dir`.